### DISCUSSION TITLE
Adding more Preset Speeds

### DISCUSSION THREAD

**Discussion by Lgxprod1** [cagnulein/qdomyos-zwift#2180] [General]
My treadmill does not have preset speeds like 1mph 2 mph, all the way to 10mph so I have been relying on QZ for the preset speed and just tapping it to change my treadmill to that chosen speed. It has been SUPER helpful!!! 

I was hoping you can add 5 more preset speeds from the current 5 so there will be a total of 10 preset speeds. This will make it a lot easier for me to switch between a variety of speeds especially during my longer/speedier sessions. I am currently using the android playstore version. Thank you!

**Comment by cagnulein** [comment]
Hi, sure I can do that. Please consider to join the patreon membership if
you didn't already! https://www.patreon.com/cagnulein

Thanks

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 3 mar 2024 alle 23:42 Lgxprod1 ***@***.***> ha
scritto:
